src/tab/tab.svelte

```diff
@@ -0,0 +1,34 @@
+<script>
+  import { createEventDispatcher } from 'svelte';
+  import rippleEffect from '../_utils/ripple.js';
+  import classes from '../_utils/classes.js';
+
+  let _class = null;
+  export { _class as class };
+  export let inputClass = null;
+  export let contentClass = null;
+
+  export let value;
+  export let group = null;
+  export let disabled = false;
+  export let ripple = true;
+
+  const dispatch = createEventDispatcher();
+</script>
+
+<label class={classes('tab', _class)}>
+  <input
+    bind:group
```

Comment:
I cannot find the attribute named `group` anywhere in the [MDN docs](https://www.youtube.com/watch?v=rNr_0TUXmAA). Did you mean `name`?

Reply:
No, this is Svelte-specific. This would be the value of the currently selected radiobutton

Reply:
Nice link, btw :p

Reply:
Oops, Windows clipboard did it again 😅. Fixed link.
Not sure why Svelte did this when we have the `name` attribute, but ok

Reply:
The `name` attribute serves a different purpose. `name` defines a group of radiobuttons, while the `group` attribute contains a value of the currently selected radiobutton, the same for all radiobuttons.